1-exercises/D-do-while/exercise.js

```diff
@@ -7,9 +7,18 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+  // TODO
+  let total = 0;
+  let i = 0;
+  do {
+    if (i % 2 === 0) {
+      total += i;
+    }
+    i++;
+  } while (i < n * 2);
+  return total;
```

Comment:
Similar to Exercise B, consider applying this formula 2 * (0 + 1 + 2 + n) to simplify your code?